ansible/playbooks/roles/kubernetes_promote/handlers/main.yml

```diff
@@ -1,19 +1,17 @@
 ---
```

Comment:
In fact something similar can be done after certificates re-generation instead of container runtime service restart, but this is not the scope of your task.